pkg/scheduler/framework/runtime/framework.go

```diff
@@ -965,7 +965,7 @@ func (f *frameworkImpl) RunPermitPlugins(ctx context.Context, state *framework.C
 		if !status.IsSuccess() {
 			if status.IsUnschedulable() {
 				msg := fmt.Sprintf("rejected pod %q by permit plugin %q: %v", pod.Name, pl.Name(), status.Message())
-				klog.V(4).Infof(msg)
+				klog.V(4).InfoS(msg)
```

Comment:
You should just move the `msg` into `InfoS` and migrate to structured log.

Reply:
yes,I accept your suggestion and fixed them already, Thank you very much.